util/scrape.py

```diff
@@ -48,15 +48,18 @@ def _get_tables(self):
         table_one.extend(table_two)
         return table_one
 
-    def _build_rule_dict(self, html):
+    def _build_rules(self, html):
         """Build the tree of rules as a dictionary"""
-        rule_dict = {}
-        for i in html:
-            i = self._cleanup_string(i)
+        rules = []
+        clean_html = map(self._cleanup_string, html)
```

Comment:
Why do you iterate over html once to clean it, and then again to build the rules?